services/minespace-web/src/components/dashboard/mine/variances/VariancesTable.js

```diff
@@ -25,15 +26,16 @@ const defaultProps = {
   openViewVarianceModal: () => {},
 };
 
-export class VarianceTable extends Component {
-  getOverdueStyle = (isOverdue) => (isOverdue ? { color: errorRed } : {});
+export class VariancesTable extends Component {
+  getOverdueClassName = (isOverdue) => (isOverdue ? "color-error" : "");
```

Comment:
Having this as a function seems really odd to me. Maybe define "color-error" as a local constant and put a ternary where it's consumed? No big deal, but I don't think this is done anywhere else